src/convertToObject.js

```diff
@@ -12,7 +12,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+
+  const splitedString = sourceString.split(';');
+  const splitedProperties = splitedString.map(words => words.split(':'));
+  const trimmedProperties = splitedProperties.map(
+    word => word.map(string => string.trim())
+  );
+
+  for (const item of trimmedProperties) {
```

Comment:
Не називаємо змінні item
```suggestion
  for (const property of trimmedProperties) {
```